packages/node_modules/@webex/redux-module-conversation/src/reducer.js

```diff
@@ -45,6 +45,17 @@ export const initialState = fromJS({
   }
 });
 
+const markEditedActivity = ({activity, activities}) =>
+  activities.map((activityItem) => {
+    if (activityItem.id === activity.parent.id && activity.object.displayName) {
+      activityItem.object.displayName = activity.object.displayName;
+      return Object.assign(activityItem, activity.object.displayName, {
```

Comment:
If we have already assigned the displayName in previous line, why are we assigning it again and returning?

Reply:
@mkesavan13 Good catch! 
I have updated the code.